developer-docs-site/docs/move/move-on-aptos/gas-profiling.md

```diff
@@ -0,0 +1,129 @@
+---
+title: "Gas Profiling"
+---
+
+import CodeBlock from '@theme/CodeBlock';
+
+# Gas Profiling
+The Aptos Gas Profiler is a powerful tool that can help you understand the gas usage of Aptos transactions. Once activated, it will simulate transactions using an instrumented VM, and generate a web-based report. <a href="/gas-profiling/sample-report/index.html" target="_blank">[Sample]</a>
+
+The gas profiler can also double as a debugger since the report also includes a full execution trace.
+
+## Using the Gas Profiler
+The gas profiler can be invoked by appending the `--profile-gas` option to Aptos CLI’s `move publish`, `move run` or `move run-script` commands.
+
+Here is an example using the `hello_blockchain` package. First, `cd` into the package directory.
+
+```bash
+$ cd aptos-move/move-examples/hello_blockchain
+```
+
+Then, we can simulate module publishing with the extra option `--profile-gas`. Notice that you do need to have your CLI profile set up properly and bind the named addresses correctly.
```

Comment:
This seems like a common got-cha.  Can you make it in its own line and bold the words on "CLI profile set up"?